### PR TITLE
Add live arg to listGroupIds

### PR DESCRIPTION
**Pull request by Powersource**
for https://github.com/ssbc/ssb-tribes2/issues/23

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,13 +18,14 @@
     "pull-stream": "^3.6.14",
     "private-group-spec": "^2.1.1",
     "ssb-bfe": "^3.7.0",
-    "ssb-keyring": "^3.0.0",
+    "ssb-keyring": "https://gitlab.com/powersource/ssb-keyring/-/archive/live-list/ssb-keyring-live-list.tar",
```
Comment (Powersource) [inline package.json:21]: need to merge and publish https://gitlab.com/ahau/lib/ssb-keyring/-/merge_requests/19

**Comment by Powersource** [comment]
major release since it moves the arg position of the cb

File: README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ on the `sbot.box2` namespace:
   - `key` must be a buffer. The key can then be used as a "recp" to encrypt messages to the group. Note that the keys are not persisted in this module.
   - `scheme` _String_ - scheme of that encryption key (optional, there is only one option at the moment which we default to)
   - `root` _MessageId_ the id of the `group/init` message
-- `listGroupIds(cb) => [groupIds]`: Lists all groupIds whose messages you're able to decrypt. Returns a promise if cb isn't provided.
+- `listGroupIds({ live }, cb) => [groupIds]`: Lists all groupIds whose messages you're able to decrypt. If `live` is true then the cb instead returns a pull stream with all previous and future group ids. Returns a promise if cb isn't provided.
```
Comment (staltz) [inline README.md:75]: Like my comment in the ssb-keyring PR, this API should also return the same consistent type, i.e. always pull-stream.

Comment (Powersource) [reply README.md:75]: yeah actually using this in tribes2 felt very weird

Comment (Powersource) [reply README.md]: done

**Review by staltz — APPROVED** [review]
Looks good! Remember this is a breaking change, so publish a new major ver.

**Pull request merged**